src/js/src/actions.data.js

```diff
@@ -10,6 +10,10 @@ import {
 export const START_OVERPASS_REQUEST = 'START_OVERPASS_REQUEST';
 export const FAIL_OVERPASS_REQUEST = 'FAIL_OVERPASS_REQUEST';
 export const COMPLETE_OVERPASS_REQUEST = 'COMPLETE_OVERPASS_REQUEST';
+export const START_GEOCODER_SEARCH = 'START_GEOCODER_SEARCH';
+export const FAIL_GEOCODER_SEARCH = 'FAIL_GEOCODER_SEARCH';
+export const COMPLETE_GEOCODER_SEARCH = 'COMPLETE_GEOCODER_SEARCH';
+export const SELECT_GEOCODER_RESULT = 'SELECT_GEOCODER_RESULT';
```

Comment:
Just realized that all the changes in this file may be unused now that I've moved everything into a geocoder actions file, so I'll push a fixup to drop them.